src/AtomMode.ts

```diff
@@ -141,3 +142,11 @@ export function drawAtom(thisAtom: AtomNode, color: string, currentAtom: Boolean
     }
     ctx.stroke();
 }
+
+/**
+ * When the checkbox for showing all bounding boxes is checked redraws the canvas showing the boxes.
+ */
+function checkBoxRedraw() {
+    ctx.clearRect(0, 0, canvas.width, canvas.height);
+    redrawCut(tree.sheet, offset);
```

Comment:
Should probably be its own function.

Reply:
Would you like me to do this now or is it ok to wait until after milestone 5?